app/controllers/network_services_controller.go

```diff
@@ -49,6 +49,8 @@ const (
 	svcHairpinAnnotation   = "kube-router.io/service.hairpin"
 	svcLocalAnnotation     = "kube-router.io/service.local"
 	svcSkipLbIpsAnnotation = "kube-router.io/service.skiplbips"
+
+	LeaderElectionRecordAnnotationKey = "control-plane.alpha.kubernetes.io/leader"
```

Comment:
this is a constant copied from client-go, I'll use the one in client-go once the vendor situation is fixed